### DISCUSSION TITLE
Firefly III Chrome Extension "Ecosystem"

### DISCUSSION THREAD

**Discussion by bradsk88** [firefly-iii/firefly-iii#7510] [General]
Hey all, I've been working on a set of chrome extensions so that I could easily scrape transactions from my financial institutions without having to involve a third party and without having to deal with 2-factor-authentication **every single time** (due to distributed architecture of services like Mint).

I've reached a point where it's working well for me personally, but is not quite polished enough for general consumption.

Thought I'd share it with you all to see if anyone is interested in leveraging a project like that.

https://github.com/firefly-iii/firefly-iii/assets/2147630/09915e68-c532-410f-b10f-da2aa0f96810

Here are the project links:
- https://github.com/bradsk88/firefly-iii-chrome-extension-base
- https://github.com/bradsk88/firefly-iii-chrome-extension-hub

And attribution for some of the art I used in the vid:
- [[1]](
https://www.vecteezy.com/png/9398418-bank-clipart-design-illustration
) [[2]](
https://www.vecteezy.com/png/17805944-3d-illustration-of-finger-print
) [[3]](
https://www.vecteezy.com/png/16774415-green-check-mark-icon-png-on-transparent-background
)

Cheers!

**Comment by JC5** [comment]
This is really cool! I'm gonna plug this on [Mastodon](https://fosstodon.org/@ff3) later 😎
